Week-13/shortener-url-app/src/components/Search.js

```diff
@@ -0,0 +1,42 @@
+import React, { useRef, useState } from 'react';
+import axios from 'axios';
+
+const Search = () => {
+    let inputRef = useRef();
+    const [shortUrls, setShortUrls]= useState([]);
+    const handleClick = () => {
+        postURL(inputRef.current.value);
+    };
+
+    const postURL = (url) => {
+        axios.post(`https://api.shrtco.de/v2/shorten?url=${url}`).then((response) => {
+            if (response?.data?.result) {
+                setShortUrls([...shortUrls, {originalURL: response.data.result.original_link, shortURL: response.data.result.full_short_link}]);
+            }
+            axios.get(`https://api.shrtco.de/v2/info?code=${response.data.result.code}`).then((res) => {
+                console.log(res);
+            }).catch((err) => console.error(err));
+        }).catch((err) => console.error(err));
+    };
+
+    console.log(shortUrls);
```

Comment:
Kindly remove the `console.log` statement.

Reply:
Done